src/Compilers/CSharp/Portable/Symbols/SymbolWithAnnotations.cs

```diff
@@ -15,9 +15,95 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
     /// A simple class that combines a single type symbol with annotations
     /// </summary>
     [DebuggerDisplay("{GetDebuggerDisplay(), nq}")]
-    internal abstract class TypeSymbolWithAnnotations : IFormattable
+    internal readonly struct TypeSymbolWithAnnotations : IFormattable
```

Comment:
Consider renaming this file to match the type.

Reply:
Yes. I'll rename the file in a separate PR.

---
In reply to: [207053923](https://github.com/dotnet/roslyn/pull/28943#discussion_r207053923) [](ancestors = 207053923)